README.md

```diff
@@ -83,6 +83,48 @@ The following packages are suggested:
 [phpDox]: http://phpdox.de
 [Sami]: https://github.com/FriendsOfPHP/sami
 
+## Usage
+
+This is a simple metapackage, but still can be used in different ways.
+
+#### Global install
+
+This is by far the easiest and most convenient method. Simply having all the tools available system wide.
+
+The following script will install a system wide Composer for you, including the QA tools.
+
+```bash
+bash <(curl -S https://raw.githubusercontent.com/DealerDirect/php-qa-tools/master/bin/install.sh)
```

Comment:
Nice find! Will do a similar thing 👍 

I totally agree on the last bit... little out of scope for this pull request. Sound like a 'guide'. Let's discuss this post daily and consider enforcing it.